examples/todoapp/lib/lib_router/models/route_model.dart

```diff
@@ -9,21 +9,36 @@ import '../../lib_permissions/models/route_permissions.dart';
 import 'routes_path.dart';
 
 enum RouteModel {
-  dashboard(
-    pathName: RoutesPath.dashboard,
-    fullPath: '/dashboard',
-    permissionName: RoutePermissions.dashboard,
+  todoDetails(
+    pathName: RoutesPath.todoDetails,
+    fullPath: '/todo-list/details/:id',
+    permissionName: RoutePermissions.todoDetails,
+  ),
+  todoCreate(
+    pathName: RoutesPath.todoCreate,
+    fullPath: '/todo-list/create',
```

Comment:
Should we remove the `todo-list` path here?  Shouldn't it be inherited from the parent route? I think the end result here will be `/todo-list/todo-list/create` for example. Or I'm wrong

Reply:
Nope ... `fullPath` really means `fullPath` :)